pkg/logger/http.go

```diff
@@ -93,39 +73,39 @@ type HTTPRequestSummary struct {
 // HTTPRequest logs a summary of an HTTP request
 // service is the name of the service the request is being made to
 func (l *logger) HTTPRequest(service string, duration time.Duration, req *http.Request, res *http.Response) {
-	reqHeaders := []string{}
-	for header, values := range req.Header {
-		for _, value := range values {
-			reqHeaders = append(reqHeaders, header+": "+value)
-		}
-	}
+	summary := createRequestSummary(service, duration, req, res)
 
-	resHeaders := []string{}
-	for header, values := range res.Header {
-		for _, value := range values {
-			resHeaders = append(resHeaders, header+": "+value)
-		}
+	if res.StatusCode >= 500 {
+		l.Warn("request summary", summary)
```

Comment:
what do you think about this one being an error log?

i think 400s being info is good because sometimes that is valid for us (e.g. check if a file exists and handle the 404).

but a 500 means something definitely went wrong and we want that being surfaced as an error?